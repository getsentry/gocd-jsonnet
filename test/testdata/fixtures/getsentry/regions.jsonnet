local getsentry = import '../../../../libs/getsentry.libsonnet';

{
  all_regions: getsentry.regions + getsentry.test_regions,
}
