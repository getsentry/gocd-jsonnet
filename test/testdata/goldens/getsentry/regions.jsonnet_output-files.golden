{
   "all_regions": [
      "s4s",
      "us",
      "customer-1",
      "customer-2",
      "customer-3",
      "customer-4",
      "customer-5",
      "customer-6"
   ]
}
